random_walk.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
This library would have to be included to the build. You would have to use some package tracking system. You should commit either 1) Pipfile along with the Pipefile.lock or 2) requirements.txt

Reply:
This would be part of the repository so that once it is checked out - it is ready to be run.